connexion/spec.py

```diff
@@ -156,6 +158,9 @@ class Swagger2Specification(Specification):
     yaml_name = 'swagger.yaml'
     operation_cls = Swagger2Operation
 
+    response = requests.get('https://raw.githubusercontent.com/OAI/OpenAPI-Specification/6d17b631fff35186c495b9e7d340222e19d60a71/schemas/v2.0/schema.json')
```

Comment:
This will be a problem in deployments that do not have access to the public internet.  We deploy in locations that are isolated from the outside world, and the application will fail to start if this change is merged.  I'd suggest including the specs in the PR so they are packaged and delivered along with connexion.  That eliminates the need for network access to retrieve them at run time.

Reply:
This cannot be like that. We cannot rely on internet connection.